### PR TITLE
ebook template nav and hero

### LINKED ISSUE #389: eBook template
Original examples:
https://www.moleculardevices.com/en/assets/ebook/br/elisa
https://www.moleculardevices.com/en/assets/ebook/bpd/accelerate-vaccine-development
https://www.moleculardevices.com/en/assets/ebook/bpd/how-to-implement-colony-picking-workflows-with-automation

- [ ] nav (https://github.com/hlxsites/moleculardevices/pull/390)
- [ ] hero (https://github.com/hlxsites/moleculardevices/pull/390)
- [ ] form
- [ ] footer

### PR DESCRIPTION
**Pull request by paolomoz**
Fix #389 (nav and hero)

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/en/assets/ebook/br/elisa
- After: https://ebook--moleculardevices--hlxsites.hlx.page/en/assets/ebook/br/elisa
- Test: https://ebook--moleculardevices--hlxsites.hlx.page/en/assets/user-guide/dd/cns/axon-guide-to-electrophysiology-and-biophysics-laboratory-techniques
- Test: https://ebook--moleculardevices--hlxsites.hlx.page/en/assets/tutorials-videos/dd/img/ananda-neurite-untangled
- Test: https://ebook--moleculardevices--hlxsites.hlx.page/en/assets/ebook/dd/img/generate-application-specific-high-content-imaging-and-analysis
- Test: https://ebook--moleculardevices--hlxsites.hlx.page/en/assets/scientific-posters/dd/img/automation-of-3d-organoid-culture-workflow-with-deep-learning-based-image-analysis
- Test: https://ebook--moleculardevices--hlxsites.hlx.page/en/assets/presentations/dd/img/advantages-and-challenges-of-phenotypic-screening-in-3d-culture-including-ecm

### CODE REVIEW

**Review by andreituicu — APPROVED** [review]
1. I think in the hero the `hero-background img` should have `object-position: center`. To me, it looks closer to the original.

2. On mobile the text `ELISA explained: from basics to practical application` is missing. It is replaced by a horizontal line.

Otherwise LGTM!

**Comment by mhaack** [comment]
Please double check with the MolDev colleges but I think this should have a more generic name, not eBook. Gated pages for eBooks are just one sample use case. 

This style of pages is used mostly for gated pages before uses can download eBooks, Application Notes and other docs. But I also saw it on webinar pages.

File: templates/ebook/ebook.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
+.ebook .hamburger,
+.ebook header .mainmenu-wrapper,
+.ebook header .navbar-header > .company-links,
+.ebook main .breadcrumbs {
+  display: none;
```
Comment (mhaack) [inline templates/ebook/ebook.css:5]: This is an interesting approach. However it is best practice to have separate header and also footer docs for campaign or special pages like this. 

The default header and footer can be overwritten be meta data field already. We can extend this logic be loading a different header and footer doc based on template.

**Review by andreituicu — APPROVED** [review]
LGTM! 

I see some minor differences, but nothing critical.

**Review by andreituicu — APPROVED** [review]
LGTM! 

I see some minor differences, but nothing critical.

**Pull request merged**